python/lsst/daf/butler/registries/sqlRegistry.py

```diff
@@ -992,6 +994,42 @@ def find(self, collection, datasetType, dataId):
         else:
             return None
 
+    def getRegion(self, dataId):
+        """Get region associated with a dataId.
+
+        Parameters
+        ----------
+        dataId : `dict`
+            A `dict` of `DataUnit` name, value pairs that label the `DatasetRef`
```

Comment:
Is it confusing to call out to the `DataUnit` class docs here even though these arguments are not of that class?